Added bootorder tool and OpenCore logo, updated Windows and NixOS logos

Litarvan opened the pull request:
I updated a few logos:
  - I added an icon for the "bootorder" tool since it was the only one that was missing (except for deprecated tools).
  - I added a logo for OpenCore, the most popular Hackintosh bootloader
  - I added a logo for [netboot.xyz](https://netboot.xyz/), I used a modified version of the net badge since the official netboot.xyz logo really didn't fit.
  - I swapped the nixos and nixos-alt logos, since nixos-alt (which I made the default) was the actual official NixOS logo, while the other one was a non-official one which has a different style than all other logos (non flat and non colorful).
  - I removed the win10 and win8 logos, and made win11 the win8 one. The reason for that is that rEFInd will actually never use the win10 and win11 logos, since it has no idea what version of Windows is installed. Instead, [it simply consider that the presence of the EFI Windows bootloader means that "win8" should be used](https://sourceforge.net/p/refind/code/ci/master/tree/refind/scan.c#l568). If not, which is the case for older Windows versions, "win" will be used. This is why #44 exists, the only logo rEFInd will use is "win8" for Windows >=8. Since Windows 8 is not supported anymore, as are all versions of Windows 10 except the last one, I think it is viable to use the Windows 11 logo for this. I also replaced the "windows" logo since it was the Microsoft one.

Here is a preview of all these changes:
![screenshot_1](https://github.com/bobafetthotmail/refind-theme-regular/assets/8712146/de23c605-b73f-4880-8d3f-82b4a026b170)
![screenshot_2](https://github.com/bobafetthotmail/refind-theme-regular/assets/8712146/e2b16342-7ec2-45e8-aca9-e84536fb9938)

bobafetthotmail commented:
Hello and thanks for the time to investigate the Windows icon issue.
The icons for NixOS and OpenCore are fine.
For the Windows icons, I would prefer to keep the older windows icons so people can "choose" them by just renaming the icons (will add a paragraph in the Readme explaining what to do).
Can you please rename them to something else like say windows_8 windows_10 instead of deleting them? You can just add a commit with the change on top of the other one, I'll merge everything into one.

As for the netboot icon, from what I have been able to gather by googling, it has nothing to do with netboot.xyz or ipxe so that's a no.
Please remove it or replace it with something like this, where their official icon which has just a black outline has been put inside a light grey background so it's always readable and looks more like an icon. 
![netboot xyz1](https://github.com/bobafetthotmail/refind-theme-regular/assets/9494179/3d410211-9999-428f-bb80-4685fe39ada9)

Litarvan commented:
Hello!

You are right for the Windows icons, I thought of that too later. I added back the Windows 8 and Windows 10 icons as `os_win8.png` alts: `os_win8-alt1.png` and `os_win8-alt2.png`) I don't think naming them something like `os_win10.png` is a good idea, since it will create the same confusion that led to the #44. Since `os_win8` is the actual name rEFInd is looking for, naming them as alts of this is probably a good idea.
I also added back the Microsoft icon as `os_windows-alt.png`.

For the netboot icon, I was afraid that using the official svg would look odd. But adding a light grey background is a good idea, I replaced the icon. I struggled a bit to find a way to do it properly: I made a square to keep the same proportions as the other icons, and tried to use a shade of grey that was visible on both themes. It looks like this, tell me what you think:

![image](https://github.com/bobafetthotmail/refind-theme-regular/assets/8712146/a6d8271d-241c-427f-bdb8-b447ccceb312)
![image](https://github.com/bobafetthotmail/refind-theme-regular/assets/8712146/5f538727-8ea5-4bd6-bfac-5dc89b8c7c99)

It's not perfect but it works, tell me if you want me to made adjustments.
